<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sample_select</class>
 <widget class="QWidget" name="sample_select">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>165</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="next_step">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
    </font>
   </property>
   <property name="text">
    <string>Next Step</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Sample3</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Sample1</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Sample2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Step1:  Select test samples</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>next_step</sender>
   <signal>clicked()</signal>
   <receiver>sample_select</receiver>
   <slot>NextStep()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>next_step</sender>
   <signal>clicked()</signal>
   <receiver>sample_select</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>NextStep()</slot>
 </slots>
</ui>
